docs/notebooks/sdba.ipynb

```diff
@@ -474,55 +474,16 @@
     "\n",
```

Comment:
<p>This section's title should be changed as this is not "multi-step" anymore.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/xclim/pull/1580/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Ok, I thought that stacking variables could still count as an extra step, but granted</p><p> </p>